lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -959,7 +961,18 @@ def view_init(self, elev=None, azim=None):
         else:
             self.azim = azim
 
-    def get_proj(self):
+    def set_proj(self, proj):
+        """
+        Set the type of projection.
```

Comment:
Do you think it will be okay if I rename the flag and this setter to `proj_type`? since I will keep `get_proj`, using `set_proj` would cause confusion.